src/notifications/contracts/model-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "to": "String",
+  "type": "String",
+  "title": "String",
+  "message": "String",
+  "unread": {
+    "type": "Boolean",
+    "default": true
+  },
+  "sent": {
+    "type": "Boolean",
+    "default": false
+  },
```

Comment:
Agree with @tatianacmh, the `updatedAt` it would be great but I think we should call it "readAt" since the only thing we are going to update is the field `unread`. What do you think @jac1013 ?